python/lsst/afw/detection/utils.py

```diff
@@ -49,3 +54,51 @@ def writeFootprintAsDefects(fd, foot):
     y0:     %4d                         # Starting row
     height: %4d                         # number of rows
 }""" % (bbox.getMinX(), bbox.getWidth(), bbox.getMinY(), bbox.getHeight()), file=fd)
+
+
+def footprintsToNumpy(
```

Comment:
Needs a unit test.

Reply:
I added tests for both the footprint ID and boolean footprints and will rerun Jenkins.